README.md

```diff
@@ -29,3 +29,10 @@ The run script will build the app, dockerize it, and deploy to docker compose au
 * `docker logs -f openbrewerydb` - Shows the server logs.
 * `docker-compose stop` - Stops all containers
 * `docker-compose up` - Starts all containers, or restarts if there were changes.
+
+## Running Dependency Analysis
+We can run a dependency check analysis that will show us vulnerabilities. This command will start the analysis:
+
+`./gradlew dependencyCheckAnalysis`
```

Comment:
This should be `./gradlew dependencyCheckAnalyze`

Reply:
Yep I can change this in a bit!

Reply:
https://github.com/openbrewerydb/openbrewerydb-java-api/pull/20/commits/f1a1ce09fc29b6ee89733609d0a0443086c4fe32